### ISSUE TITLE
[Issue] [10.8.0 Beta 2]: When importing using local NFO files for Movies or TV Shows, certain external ids are missing

### ISSUE THREAD

**Issue by almightiest** [jellyfin/jellyfin#7739]
### Please describe your bug

**Metadata in Jellyfin**
_Movies_: has imdb id, missing tmdb id
_TV Shows_: has tvdb id, missing imdb id
_TV Episodes_: missing all ids

**NFO files attached**
[Archive.zip](https://github.com/jellyfin/jellyfin/files/8673490/Archive.zip)

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS: Ubuntu 20.04.5
- Virtualization: No
- Clients: Browser, Shield TV
- Browser: Chrome OS Stable 100
- FFmpeg Version: n/a
- Playback Method: n/a
- Hardware Acceleration: n/a
- Plugins: n/a
- Reverse Proxy: No
- Base URL: n/a
- Networking: local intranet
- Storage: local
```

### Please attach any screenshots here

Movie:
![image](https://user-images.githubusercontent.com/1410483/167949908-61ff2640-b02c-414c-9b34-b2ada4349026.png)

TV Show:
![image](https://user-images.githubusercontent.com/1410483/167949781-6b2b32c0-a44d-46f4-842a-1fd3dc6973d6.png)

TV Episode:
![image](https://user-images.githubusercontent.com/1410483/167949696-d9e6de84-adb0-4c53-8f2c-99390374f955.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Shadowghost** [comment]
Should be fixed by #7654

**Comment by daullmer** [comment]
Did a retest with the files you provided with 10.8. Seems to be working
